Assets/Scripts/LevelManager.cs

```diff
@@ -1,22 +1,34 @@
 ﻿using UnityEngine;
 using System.Collections;
+using UnityEngine.SceneManagement;
 
 public class LevelManager : MonoBehaviour {
 
-    public float autoLoadNextLevelAfter;
+    [SerializeField]
+    private bool shouldAutoload;
+    [SerializeField]
+    private float autoloadNextLevelAfter;
 
     void Start()
     {
-        Invoke("LoadNextLevel", autoLoadNextLevelAfter);
+        if(shouldAutoload)
+        {
+            Invoke("LoadNextLevel", autoloadNextLevelAfter);
+        }
     }
 
     public void LoadNextLevel()
     {
-        Application.LoadLevel(Application.loadedLevel + 1);
+        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex + 1);
     }
 
     public void RequestQuit()
     {
         Application.Quit();
     }
+
+    public void GameOver()
+    {
+        SceneManager.LoadScene("Game_Over");
```

Comment:
As you already said.The "Game_Over" should be made into a constant or maybe we need some sort of Scene enumeration to keep a list of the scenes? The enum would probably be clearer as well since we'd likely then have:
SceneList.GameOverScene or something similar ?